README.md

```diff
@@ -222,6 +224,26 @@ WebServiceClient client = new WebServiceClient.Builder(6, "ABCD567890").build();
 System.out.println(client.insights(request));
 ```
 
+### Report Transaction
```

Comment:
It'd be nice to have some documentation too about what this is and maybe a link to the dev docs. Pat added some for the PHP API. I don't know if we need as much as he added, but just a short summary would be good.

Reply:
I've stolen some of Pat's work.